pkg/operator/ceph/csi/spec.go

```diff
@@ -178,9 +179,10 @@ var (
 )
 
 const (
-	KubeMinMajor                     = "1"
-	KubeMinVerForOIDCTokenProjection = "20"
-	kubeMaxVerForBeta1csiDriver      = "21"
+	KubeMinMajor                       = "1"
+	KubeMinVerForOIDCTokenProjection   = "20"
+	kubeMaxVerForBeta1csiDriver        = "21"
+	KubeMinVerForCSIDriverSeLinuxMount = "25"
```

Comment:
no need for this to be a public const
```suggestion
	kubeMinVerForCSIDriverSeLinuxMount = "25"
```

Reply:
made this and few other variable private